src/unit_tests/shared/test_json_op.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+#include <string.h>
+#include <unistd.h>
+
+
+#include "../../headers/json_op.h"
+#include "../wrappers/externals/cJSON/cJSON_wrappers.h"
+#include "../wrappers/wazuh/shared/file_op_wrappers.h"
+#include "../wrappers/wazuh/shared/debug_op_wrappers.h"
+#include "../wrappers/libc/stdio_wrappers.h"
+#include "../wrappers/common.h"
+
+#define MAX_LENGHT_BUFFER_TEST 200
+
+static void test_json_fread_one(void **state) {
+    (void) *state;
+
+    char * path = (char *)malloc(sizeof(char)*MAX_LENGHT_BUFFER_TEST);
+    strcpy(path, "/home/test");
+    expect_w_get_file_content(NULL);
+    expect_string(__wrap__mdebug1, formatted_msg, "Cannot get the content of the file: /home/test");
+    assert_ptr_equal(json_fread(path, 1), NULL);
+
+    free(path);
+}
+
+static void test_json_fread_two(void **state) {
+    (void) *state;
```

Comment:
I assume this was to silence the compiler (been there)

Reply:
Maybe [this](https://stackoverflow.com/a/3125889/13272169) is helpful.